src/app/components/datetime-picker/datetime-picker.component.html

```diff
@@ -27,9 +28,9 @@
     <ion-content>
       <ion-datetime
         [value]="dateTime"
-        (ionChange)="updateTempDateTime($event)"
-        [max]="maxDate"
+        (ionChange)="updateTempDateTime($event.detail.value)"
```

Comment:
Jeg vet det ikke er du som har laget updateTempDateTime(), men kunne du i stedet ha brukt to-veis binding?